lib/Entity/Video.php

```diff
@@ -397,6 +397,12 @@ public function setKeywords($keywords)
      */
     public function getStills()
     {
+        //sorting preview's images from smallest to biggest
+        usort($this->stills, function (array $item1, array $item2) {
+            if (isset($item1['dimension']['height']) && isset($item2['dimension']['height'])) {
```

Comment:
`isset()` accepts multiple parameters, and when used with multiple parameters will only return true if all parameters passed exist